prog/github/repository_nexus.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+require_relative "../../lib/util"
+
+class Prog::Github::RepositoryNexus < Prog::Base
```

Comment:
I used `Prog::Github::RepositoryNexus` instead of `Prog::Github::GithubRepositoryNexus`. This is different from, for example, `Prog::Postgres::PostgresServerNexus`. What do you think about @byucesoy?

Reply:
I added a comment about this. It was part of the review I just published.